.travis.yml

```diff
@@ -24,7 +24,7 @@ addons:
 #  - "sh -e /etc/init.d/xvfb start"
 
 env:
-  - VERSION="11.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="11.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" TESTS="1"
```

Comment:
Isn't `LINT_CHECK=0` default?

Reply:
Yes, Maybe the first author is based on our template:
https://github.com/OCA/maintainer-quality-tools/blob/aafbddb5aed3dd1155609b1d16f982d832941c44/sample_files/.travis.yml#L34

Do you recommend us removed it?

Reply:
Totally agree, I was more curious - not requesting change